test/remote/gateways/remote_stripe_test.rb

```diff
@@ -10,6 +10,23 @@ def setup
     @new_credit_card = credit_card('5105105105105100')
     @debit_card = credit_card('4000056655665556')
 
+    @account = fixtures(:stripe_destination)[:stripe_user_id]
```

Comment:
A couple of comments for consideration here:

1) This code is in a setup method, which means that is going to run on every test call, so you are going end up checking something that is not needed on a bunch of tests.

2) this chunk of code needs to be defined in a method that gives it meaning and structure.
Something like this =>
https://gist.github.com/28820c5fa3449f0256dac766c9a80034

3) that method should only be called on the failing test so that check only impacts on that specific test.

4) Check the enumerable module on Ruby to deal with collections so you don't need to do a 'for' some basic operations.